src/__tests__/index.test.js

```diff
@@ -23,7 +114,7 @@ describe('index', function () {
     'https://fake.adobeassets.com:443/launch-ENabc123-development.min.js';
 
   beforeEach(function () {
-    window._satellite = {
+    satelliteMock = {
```

Comment:
`satellite` is now provided to the inner inject closure (you'll see this in index.js), so the tests don't assume window._satellite for tests, we pass it as a mock.

index.js will still place _satellite on the window for production.